collect_app/src/main/java/org/odk/collect/android/widgets/utilities/GeoWidgetUtils.java

```diff
@@ -65,7 +65,7 @@ public static String getGeoPolyAnswerToDisplay(String answer) {
 
     static double getAccuracyThreshold(QuestionDef questionDef) {
         // Determine the accuracy threshold to use.
-        String acc = questionDef.getAdditionalAttribute(null, ActivityGeoDataRequester.ACCURACY_THRESHOLD);
+        String acc = questionDef.getAdditionalAttribute(null, "accuracyThreshold");
```

Comment:
This string only happened to be the same before. Could definitely pull it out as a constant, but we're only using the attribute in non-test code here.